functions/exercise2/script.js

```diff
@@ -1 +1,16 @@
 "use strict";
+
+function isPalindrome(phrease) {
+    var isPhreasePalindrome = phrease.toLowerCase().split("").reverse().join("");
+    if (isPhreasePalindrome === "") {
+        throw new TypeError("Parameter must contain at least one letter - not use white space");
+    }
+    if (isPhreasePalindrome === " ") {
+        throw new TypeError("Parameter must contain at least one letter - not use white space");
+    }
+    if (isPhreasePalindrome === phrease.toLowerCase()) {
```

Comment:
why can't you simply return isPhreasePalindrome === phrease.toLowerCase() ? It shortens 5 lines to 1.